cves/kernel/CVE-2017-7487.yml

```diff
@@ -49,13 +49,13 @@ description_instructions: |
 
   Try to still be specific in your description, but remove project-specific
   stuff. Remove references to versions, specific filenames, and other jargon
-  that outsiders to this project would not understand. Technology like "regular
-  expressions" is fine, and security phrases like "invalid write" are fine to
+  that outsiders to this project would not understand. Technology like 'regular
+  expressions' is fine, and security phrases like 'invalid write' are fine to
   keep too.
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: 'This vulnerability was caused by mishandling of a reference counter which allowed for resources to be freed then accessed causing a crash.'
```

Comment:
Kind of a run on sentence. Maybe split it into two sentences? (ex. "...resources to be freed. If then accessed, it would cause the system to crash."

Reply:
Fixed